src/SurveyService.Validation/Group/CreateGroupRequestValidator.cs

```diff
@@ -0,0 +1,40 @@
+using FluentValidation;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Group;
+using LT.DigitalOffice.SurveyService.Validation.Group.Interfaces;
+using LT.DigitalOffice.SurveyService.Validation.Question.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.SurveyService.Validation.Group;
+
+public class CreateGroupRequestValidator : AbstractValidator<CreateGroupRequest>, ICreateGroupRequestValidator
+{
+  public CreateGroupRequestValidator(
+    [FromServices] ICreateGroupQuestionRequestValidator createGroupQuestionRequestValidator)
```

Comment:
don't need "[FromServices]" here; it's not wrong, but unnecessary

Reply:
Fixed